Auctions/AppendAndReadBatchRedis.fs

```diff
@@ -12,24 +12,24 @@ type AppendAndReadBatchRedis(connStr:string) =
   let hashCreate (batch : IBatch) command = 
     let id = Guid.NewGuid().ToString("N")
     let entries = mapToHashEntries command
-    batch.HashSetAsync(redisKey id, entries |> List.toArray) |> ignore
+    batch.HashSetAsync(RedisKey.op_Implicit id, entries |> List.toArray) |> ignore
```

Comment:
Maybe you can use `implicit` from F#+ have you tried it?

Reply:
👍